README.md

```diff
@@ -144,6 +144,28 @@ vars:
 vars:
   hubspot__pass_through_all_columns: true # default is false
 ```
+
+### Adding property label
+For `property_hs_*` columns, you can enable the corresponding, human-readable `property_option`.`label` to be included in the staging models. 
+- **Note** you cannot bring in labels if using `hubspot__pass_through_all_columns: true`.` 
+- To enable labels for a given property, set the property attribute `use_property_label: true`, using the below format.
+
+```yml
+vars:
+  hubspot__ticket_pass_through_columns:
+    - name: "property_hs_fieldname"
+      alias: "fieldname"
+      use_property_label: true
+```
+  Alternatively, you can enable labels for all passthrough properties by using variable `hubspot__enable_all_property_labels: true`, formatted like the below example. 
+
+```yml
+vars:
+  hubspot__enable_all_property_labels: true
+  hubspot__ticket_pass_through_columns:
+    - name: "property_hs_fieldname1"
+    - name: "property_hs_fieldname2"
+```
```

Comment:
If we are not allowing this to be used when `hubspot__pass_through_all_columns: true` is used then we should call that out here.